std/bitmanip.d

```diff
@@ -3536,6 +3537,7 @@ unittest
         T     = The integral type to convert the first $(D T.sizeof) bytes to.
         endianness = The endianness to write the bytes in.
         range = The range to append to.
+        value = The value to append.
```

Comment:
Looking at the output, the parameter names are not highlighted in the same way as symbol names are so this might be a unnecessary change for parameter descriptions.